corehq/apps/hqwebapp/templates/hqwebapp/includes/modal_report_issue.html

```diff
@@ -1,150 +1,167 @@
 {% load i18n %}
 {% load hq_shared_tags %}
 {% if request.user.is_authenticated %}
-    <div class="modal fade" id="modalReportIssue">
-        <div class="modal-dialog">
-            <form id="hqwebapp-bugReportForm"
-                  action="{% url "bug_report" %}"
-                  method="post"
-                  enctype="multipart/form-data"
-                  role="form">
-                {% csrf_token %}
-                <input type="hidden" id="bug-report-url" name="url"/>
-                <input type="hidden" id="bug-report-username" name="username" value="{{ user.username }}"/>
-                <input type="hidden" id="bug-report-domain" name="domain" value="{{ domain }}"/>
-                <input type="hidden" id="bug-report-app_id" name="app_id" value="{{ app.id }}"/>
-            <div class="modal-content">
-                <div class="modal-header">
-                    <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">{% trans 'Close' %}</span></button>
-                    <h4 class="modal-title">{% trans "Report an Issue with CommCare HQ" %}</h4>
-                    <br/>
-                    <p>
-                        <b>{% trans "Please submit this report from page where issue is occurring" %}</b>
-                    </p>
+<div class="modal fade" id="conditional-alert-warning-modal" role="dialog">
```

Comment:
Why is this in `modal_report_issue.html` now? `conditional_alert_save.html` was a reasonable place for it.

Reply:
To get the html code for popup rendered beforehand. If I am putting it inside conditional_alert_save.html file it is not rendered completely at the moment I want it to show.